test/e2e/install-codeflare-sdk.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# go to codeflare-sdk folder and install codeflare-sdk
+cd ..
+
+# Install Poetry and configure virtualenvs
+pip install poetry
+poetry config virtualenvs.create false
+
+cd codeflare-sdk
+# Clone the CodeFlare SDK repository
+git clone --branch main https://github.com/project-codeflare/codeflare-sdk.git
```

Comment:
This should ideally be parameterized, but it'll probably be adapted in the context of project-codeflare/codeflare-sdk#306.

@sutaakar @ChristianZaccaria WDYT? Should we merge this PR as is and iterate on the SDK PR?

Reply:
Fine for me.

Reply:
/lgtm for merge.